server/config.js

```diff
@@ -32,7 +32,7 @@ module.exports = {
       agent: {
         maxSockets: 100,
         maxFreeSockets: 10,
-        freeSocketTimeout: 30000,
+        freeSocketTimeout: 10000,
```

Comment:
apologies, this isn't what I meant.  I meant dropping the `response` and `deadline` timeouts rather than the agent timeouts.  It is the `deadline` one that will affect retries, since if superagent doesn't get a response in that time it will retry.

Reply:
See https://github.com/ministryofjustice/new-nomis-ui/blob/master/server/api/oauthEnabledClient.js#L58 for equivalent in new nomis.  It was slightly tricker in new nomis, since we shouldn't really retry `POST` requests as it could cause duplicate data.  I guess ideally we should have defined a timeout for get and timeout for post, but we have one at present and then divide it by three for requests that have retries...

Reply:
thanks, dropped the deadline timeout to 10 seconds and reverted freesockettime